src/client.rs

```diff
@@ -95,6 +95,8 @@ impl ClientOram {
 
             for i in 0..BUCKET_SIZE {
                 for (j, data_item) in elts.iter().enumerate() {
+                    // Because dummies are not in the position map, they don't
+                    // match the condition.
```

Comment:
> There should be no concept of dummies here. data_item should not be a dummy value. The if let Some(...) is pure Rust overhead needed to unwrap the option returned by the get.

I added this comment since I took time to figure out why weren't dummies (previously read from the tree) selected.